src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -41,6 +41,7 @@ trait RedisTrait
         'redis_sentinel' => null,
         'dbindex' => 0,
         'failover' => 'none',
+        'ssl' => null
```

Comment:
Now I'm wondering about the `array_diff_key` around L302, does adding this remove the key for Predis?

Also, on this line, please as a trailing comma and the following comment:
`// see https://php.net/context.ssl`, that'd be enough as doc

Reply:
It does remove the option for predis

Reply:
Fixed